pages/AboutPage.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { View, Text, StyleSheet, TouchableOpacity, Linking } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import LinearGradient from 'react-native-linear-gradient';
+import Icon from 'react-native-vector-icons/Ionicons';
+import { getWMBRLogoSVG } from '../utils/WMBRLogo';
+import { SvgXml } from 'react-native-svg';
+
+export default function AboutPage() {
+  const openWebsite = () => {
+    const url = 'https://wmbr.org';
+    Linking.openURL(url).catch(() => {});
+  };
+
+  const openInstagram = () => {
+    const url = 'https://www.instagram.com/wmbrfm';
+    Linking.openURL(url).catch(() => {});
+  };
+
+  const openTwitter = () => {
+    const url = 'https://x.com/wmbr';
+    Linking.openURL(url).catch(() => {});
+  };
+
+  const openFacebook = () => {
+    const url = 'https://www.facebook.com/wmbrfm';
+    Linking.openURL(url).catch(() => {});
+  };
+
+  const openMastodon = () => {
+    const url = 'https://mastodon.mit.edu/@wmbr';
+    Linking.openURL(url).catch(() => {});
+  }
+
+  const openProgramGuide = () => {
+    const url = 'https://wmbr.org/WMBR_ProgramGuide.pdf';
+    Linking.openURL(url).catch(() => {});
+  };
+
+  const openLink = (url: string) => Linking.openURL(url).catch(() => {});
```

Comment:
we want these to all be static between renders or it'll trigger re-renders of children, but we can just pull them all up a level since nothing in here depends on component data (afaict)